pkg/volume/portworx/portworx_util.go

```diff
@@ -272,3 +297,7 @@ func (util *PortworxVolumeUtil) getPortworxDriver(volumeHost volume.VolumeHost,
 
 	return volumeclient.VolumeDriver(util.portworxClient), nil
 }
+
+func capacityToGB(capacity resource.Quantity) int {
+	return int(volutil.RoundUpSize(capacity.Value(), oneGig))
+}
```

Comment:
there is already `https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/util.go#L372`

Reply:
Thanks. Fixed.